classes/02class/agile101/README.md

```diff
@@ -0,0 +1,326 @@
+# Class 02
```

Comment:
The class 2 title should be handled on the /classes/02class/README.md file. I think that since this part is dedicated to the Agile, that could be the main title of the document.
Also, it would be good to have a link to this file on the Class 02 README.

Reply:
Fixed .../02class/README.md. 
Updated title.